tests/scripts/mbedtls_test.py

```diff
@@ -90,20 +89,24 @@ def __parse(self, data_f):
         :param data_f: Data file object
         :return:
         """
-        for line in data_f:
-            line = line.strip()
+        while True:
+            line = data_f.readline().strip()
             if not line:
-                continue
+                break
```

Comment:
It looks like previously, if a line were empty, we'd skip it. Now, it appears we'd stop the whole loop if we run into a blank line. Is this the case? Should we use `continue`?

Reply:
Yes, You are probably right. But we need to distinguish between a blank line and EOF.
Previously, the loop was over the lines in the file. Now, it's an infinite loop, and we need a break condition.
Perhaps:
```suggestion
            line = data_f.readline()
            if not line:
                break
            line = line.strip()
            if not line:
                continue	            
```